src/dft/backends/mklcpu/commit.cpp

```diff
@@ -138,6 +138,20 @@ class commit_derived_impl final : public dft::detail::commit_impl<prec, dom> {
                        to_mklcpu<config_param::PLACEMENT>(config.placement));
         set_value_item(descHandle, DFTI_PACKED_FORMAT,
                        to_mklcpu<config_param::PACKED_FORMAT>(config.packed_format));
+        // Setting the workspace causes an FFT_INVALID_DESCRIPTOR.
+        if (config.workspace != config_value::ALLOW)
+            throw mkl::invalid_argument("dft/backends/mklgpu", "commit",
+                                        "MKLCPU only supports workspace set to allow");
+        // Setting the ordering causes an FFT_INVALID_DESCRIPTOR. Check that default is used:
+        if (config.ordering != dft::detail::config_value::ORDERED) {
+            throw mkl::invalid_argument("dft/backends/mklgpu", "commit",
+                                        "MKLCPU only supports ordered ordering.");
+        }
+        // Setting the transpose causes an FFT_INVALID_DESCRIPTOR. Check that default is used:
+        if (config.transpose != false) {
+            throw mkl::invalid_argument("dft/backends/mklgpu", "commit",
+                                        "MKLCPU only supports non-transposed.");
```

Comment:
`dft/backends/mklgpu` should be `dft/backends/mklcpu` in all cases.

Reply:
changed in b56bf891cee45fc2381148f854ac7d4215cfb0c0